recipes/causalml/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set name = "causalml" %}
+{% set version = "0.12.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/causalml-{{ version }}.tar.gz
+  sha256: 1a5216ee58522cce725907b005bf93232eb311929b36a20fbf8ee341b5781ebb
+
+build:
+  number: 0
+  skip: true   # [win]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - pytest
+    - python
+    - cython
+    - setuptools >=41.0.0
+    - numpy
+    - scikit-learn >=0.22.0
+    - python-graphviz
+    - graphviz
+  run:
+    - python
+    - setuptools >=41.0.0
+    - pip >=10.0
+    - numpy
+    - scipy >=1.4.1
+    - matplotlib-base
+    - pandas >=0.24.1
+    - scikit-learn >=0.22.0
+    - statsmodels >=0.9.0
+    - seaborn
+    - cython
+    - xgboost
+    - pydotplus
+    - tqdm
+    - shap >0.35.0, <0.38.1
```

Comment:
I think this pin is causing you trouble.

You are depending on versions that might be too old.

Are you able to move up the requirements.

I can see this error pop up in the osx builds.

Reply:
hey @hmaarrfk thank you so much for helping looking into this. CausalML's requirement need the `shap <0.38.1`. Maybe I could try if I can relax it.

Reply:
It is likely that there was an API change. Try unpinning it here and leaving yourself a note to see what happens.